sparrow-py/pytests/math_test.py

```diff
@@ -0,0 +1,69 @@
+"""Tests for the Kaskada query builder."""
+import random
+import sys
+
+import pandas as pd
+import pyarrow as pa
+import pytest
+from sparrow_py import Session
+from sparrow_py import Table
+from sparrow_py import math
+from io import StringIO
+
+
+@pytest.fixture
+def session() -> Session:
+    """Create a session for testing."""
+    session = Session()
+    return session
+
+@pytest.fixture
+def table_int64(session: Session) -> Table:
+    """Create an empty table for testing."""
+    schema = pa.schema(
+        [
+            pa.field("time", pa.string(), nullable=False),
+            pa.field("key", pa.string(), nullable=False),
+            pa.field("m", pa.int64()),
+            pa.field("n", pa.int64()),
+        ]
+    )
+    return Table(session, "table1", "time", "key", schema)
+
+def test_read_table(session, table_int64) -> None:
+    table_int64.add_csv('\n'.join([
+        "time,subsort,key,m,n",
+        "1996-12-19T16:39:57-08:00,0,A,5,10",
+        "1996-12-19T16:39:58-08:00,0,B,24,3",
+        "1996-12-19T16:39:59-08:00,0,A,17,6",
+        "1996-12-19T16:40:00-08:00,0,A,,9",
+        "1996-12-19T16:40:01-08:00,0,A,12,",
+        "1996-12-19T16:40:02-08:00,0,A,,"
+    ]), dtype = {
+        'm': 'Int64',
+        'n': 'Int64',
+    })
+    # parse_dates = ['time'],)
+
+    # TODO: Options for outputting to different destinations (eg., to CSV).
+    # TODO: Allow running a single expression (eg., without field names)
+
+    #result = (table_int64.m + tableint64.n).execute()
```

Comment:
I'd like to run this, but currently can't since I can't make a record nor can I execute a single column. Will fix in a later PR. For now, demonstrating that we can run a query for just the table.